package.json

```diff
@@ -20,5 +20,9 @@
   "bugs": {
     "url": "https://github.com/umap-project/Leaflet.Measurable/issues"
   },
-  "homepage": "https://github.com/umap-project/Leaflet.Measurable#readme"
+  "homepage": "https://github.com/umap-project/Leaflet.Measurable#readme",
+  "dependencies": {
```

Comment:
Sorry, I meant to also remove them from package.json. npm is a pain, and we are talking of front libs here, not server side ones.